data/records/cms-learning-resources.json

```diff
@@ -144,7 +144,7 @@
       "education"
     ],
     "license": {
-      "attribution": "MIT Open Source license"
+      "attribution": "MIT License"
```

Comment:
It would be good to take the occasion of licence value clean-up and move towards the SPDX standard license values everywhere. This will help to improve FAIR score in tools like FUJ-I. We'll need small amendments in the  Jinja templates accordingly as well. Please see #3111 for all details.

Reply:
Ah, I just saw your issue. Do you have something like this in mind on how to improve the annotation on the license attributions:
```
"license": {
    "identifier": ...
    "full_name": ...
}
```

Reply:
For simplicity, it would be good to stick to the current data model, using the same `license.attribution` field, b with values containing standardised license identifiers (such as `CC0-1.0`). And, in order to display the full name, we could have a Jinja helper function on the infrastructure side translating identifiers to the standardised full license names (such as `Creative Commons Zero v1.0 Universal`) following the SPDX nomenclature.  (We could perhaps even make a link to the license in question.) And, to be on the safe side, for any new licence string we may introduce in thefuture that the Jinja helper function does not recognise, we would display just the text.

Since we are using only a handful of licenses, this approach should do, and has the merit of keeping the same data model.